web/modules/custom/joinup_test/tests/src/Kernel/JoinupKernelTestBase.php

```diff
@@ -37,6 +37,14 @@ abstract class JoinupKernelTestBase extends RdfKernelTestBase {
     'rdf_entity_test',
   ];
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function bootEnvironment() {
+    parent::bootEnvironment();
+    $this->setUpSparql();
+  }
+
```

Comment:
I simply followed your example for this one. If you think this belongs to RdfKernelTestBase, then why don't all the rest of the tests extend the KernelTestBase and not the RdfKernelTestBase since all of them are using the rdf_entity module anyway.

Reply:
Because the `RdfKernelTestBase` is setting up the SPARQL connection. Now it is being set up twice.